server/game/cards/01-Core/MobiusScroll.js

```diff
@@ -9,14 +9,13 @@ class MobiusScroll extends Card {
                 mode: 'upTo',
                 numCards: 2,
                 gameAction: ability.actions.archive(context => ({
-                    target: [context.source].concat(context.target),
-                    owner: true
```

Comment:
this is the default, so I don't think you need to make this change

Reply:
I removed `owner: true` because it is the default